doc/devel/contributing.rst

```diff
@@ -19,6 +19,59 @@ is expected to abide by our
 Below, you can find a number of ways to contribute, and how to connect with the
 Matplotlib community.
 
+.. _start-contributing:
+
+Getting started with contributing?
+==================================
+
+There is no pre-defined pathway for new contributors; you can look at existing
+issue and pull request discussions, and follow the discussions during code
+review. Or you can deep-dive into a subset of code-base to understand what is
+going on.
+
+There are a few typical new contributor profiles:
+
+* **You are a Matplotlib user, and you see a bug, a potential improvement, or
+  something that annoys you, and you can fix it.**
+
+  This is how a lot of people approach open-source development. You can search
+  our issue tracker for an existing issue that describes your problem or open a
+  new issue to inform us of the problem you observed and discuss the best
+  approach to fix it. If your contributions would not be captured on GitHub
+  (social media, communication, educational content), you can also reach out to
+  us on gitter_, `Discourse <https://discourse.matplotlib.org/>`__ or attend any
+  of our `community meetings <https://scientific-python.org/calendars>`__.
+
+* **You are not a regular Matplotlib user but a domain expert: you know about
+  visualization, 3D plotting, design, technical writing, statistics, or some
+  other field where Matplotlib could be improved.**
```

Comment:
```suggestion
  other field where Matplotlib can be improved.**
```

Reply:
I find the existing text to read better.